docs/content/en/docs/clustermgmt/storage/vsphere-storage.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "vSphere storage"
+linkTitle: "vSphere storage"
+weight: 80
+date: 2017-01-05
+description: >
+  Managing storage on vSphere
+---
+
+EKS Anywhere clusters running on vSphere can leverage the vSphere Container Storage Plug-in (also called the vSphere CSI Driver) for dynamic provisioning of persistent storage volumes on vSphere storage infrastructure. The CSI Driver integrates with the Cloud Native Storage (CNS) component in vCenter for the purpose of volume provisioning via vSAN, attaching and detaching volumes to/from VMs, mounting, formatting, and unmounting volumes on/from pods, snapshots, cloning, dynamic volume expansion, etc.
+
+### vSphere CSI Driver Deprecation
+
+EKS Anywhere versions prior to `v0.16.0` supported the installation and management of the vSphere CSI Driver in EKS-A clusters. The CSI management components in EKS-A included a Kubernetes CSI controller Deployment, a node-driver-registrar DaemonSet, a default Storage Class, and a number of related Secrets and RBAC entities.
+
+In EKS-A version `v0.16.0`, the CSI driver feature was deprecated as part of cluster creation and has been completely removed in version `v0.17.0`. However, you may self-manage this deployment to make use of the storage options provided by vSphere in a Kubernetes-native way. 
+
+### CSI Driver Cleanup for Upgrades
```

Comment:
This is an optional step right? Users don't need to cleanup the CSI driver if they need it? It just won't be managed by EKS-A?

Reply:
Yes but it most likely will not be working as expected as it's also a very old version that won't be compatible with the Kubernetes versions we support. So I think it's prudent to recommend the cleanup and self-installation.